### DISCUSSION TITLE
Installation of the latest registry on K8s

### DISCUSSION THREAD

**Discussion by darrellme** [Apicurio/apicurio-registry#2503] [Q&A]
Hi

What's the correct way of installing the apicurio registry on a non-openshift K8s cluster. I've tried following the instructions in the OperatorHub.io page (https://operatorhub.io/operator/apicurio-registry), but it looks like the K8s operator mentioned here is old (the operator hasn't been updated in the last 10 months), and therefore it installs version 2.0.0.Final. How can I get the latest version installed, and/or how are upgrades performed?

Thanks

Darrell Merryweather

**Comment by EricWittmann** [comment]
We're currently working on updates to the Operator.  As you have observed it hasn't been refreshed in awhile.  As part of that work hopefully we can make sure it works in k8s not just OpenShift.  That is certainly the goal!

@jsenko may have more to add.

**Reply by darrellme** [reply]
Thanks for the reply. In the meantime, I've created a helm chart that will install the latest apicurio on K8s, without the operator. This appears to be suiting our needs currently.
